README.md

```diff
@@ -1,12 +1,10 @@
 # ark_automate
 
 [![GitHub stars](https://img.shields.io/github/stars/bptlab/ark_automate)](https://github.com/bptlab/ark_automate)
```

Comment:
In my opinion we should only use badges for additional information not necessarily visible at first glance. Please think about what can be seen by scrolling up and what is additional information that might be of interest when on the main page of this projects repo

Reply:
Daniel said nothing about this. What do you think, @SanJSp?
I wouldn't mind the few badges. I think it's quite attractive that way.